MCprep_addon/MCprep_resources/optimize_scene.py

```diff
@@ -1,58 +1,72 @@
+# ##### BEGIN GPL LICENSE BLOCK #####
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software Foundation,
+#  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ##### END GPL LICENSE BLOCK #####
+
 import bpy
 from .. import util
 
-
-# ---------------------------------------------------------------------------- #
-#                   Optimizer code by StandingPad Animations                   #
-# ---------------------------------------------------------------------------- #
 class MCprepOptimizerProperties(bpy.types.PropertyGroup):
 
-	def TimeInScene(self, context):
+	def scene_brightness(self, context):
 		itms = [
-        ("Day", "Day time", ""), 
-        ("Night", "Night time", "")
+        ("bright", "scene is bright", ""), 
```

Comment:
If a user can hove rover to get tooltip text, then it'd be good to fill out that third string; something a bit more verbose like for dark: "Use settings that are optimized for dark scenes\nlike night time, interiors, or caves."